### ISSUE TITLE
add rssynergia to PyPI

### ISSUE THREAD

**Issue by bruhwile** [radiasoft/devops#50]
https://github.com/radiasoft/rssynergia

We have users of the Jupyter server, who need this library for use of our IPython notebooks.
We need them to be able to do 'pip install' and 'pip install --upgrade'

**Comment by robnagler** [comment]
Added rssynergia to jupyter.radiasoft.org customizer (using pip install through git) so there's less of a need for this. We do want it on PyPI, but I would prefer to wait on this work until we have external users.

You can always install this way:

```sh
pip install git+git://github.com/radiasoft/rssynergia.git@master
```

**Comment by bruhwile** [comment]
Having pip install work via git+git is fine for now.
I suggest labeling this issue as low priority and putting it off for a while.

**Comment by robnagler** [comment]
pip is sufficient, and we'll build beamsim automatically soon enough.
